lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3681,6 +3698,14 @@ namespace {
       if (forceClassMethod && decl->hasRelatedResultType())
         return nullptr;
 
+      // Hack: avoid importing methods named "print" that aren't available in
+      // the current version of Swift. We'd rather just let the user use
+      // Swift.print in that case.
+      if (!isActiveSwiftVersion() &&
+          isPrintLikeMethod(importedName.getDeclName(), dc)) {
```

Comment:
Do you see this coming up for more names, or is print a particularly pathological case?